src/config.ts

```diff
@@ -78,18 +86,22 @@ export const CONFIG = {
         },
       },
     },
-    title: {
+    descriptionValidation: {
       fullPrompt: {
         row: 16,
         col: 2,
       },
-      preferGeneratedValues: {
-        row: 19,
-        col: 2,
+      maxRetries: {
```

Comment:
see above, I would be more in favor of removing this altogether

Reply:
+1